app/models/dashboard.rb

```diff
@@ -0,0 +1,46 @@
+# :reek:UtilityFunction { enabled: false }
+class Dashboard
+  include ActiveModel::Model
+
+  # :reek:FeatureEnvy { enabled: false }
+  def initialize
+    slide_table = Slide.arel_table
+    @summary ||= Slide.select([slide_table[:page_view].sum.as('page_view'), slide_table[:download_count].sum.as('download_count'), slide_table[:embedded_view].sum.as('embedded_view')]).all[0]
```

Comment:
__[RuboCop]__

Metrics/LineLength: Line is too long. [191/160]


<!-- Comment by [SideCI](https://sideci.com) -->